MIPS/mip-0005-mip-upgrade.md

```diff
@@ -0,0 +1,315 @@
+- The consensus mechanism
+- Block validity rules
+- Proof system
+
+**Economics MIPs** would consider Mina's financial parameters, the levels and sources of funding to support development, including how a decentralized treasury should be configured and managed. For example:
+- Mina tokenomics
+- Transaction fees
+- Treasury management
+
+**Governance MIPs** would consider the rules and processes for making changes to the protocol and funding its development, and how they can be improved over time. For example:
+- MIP process
+- MIP roles and processes
+- On-Chain Voting
+
+These categories will need to be clearly defined, specifying their parameters and what kinds of major changes could be made.
+
+### New Roles and Responsibilities
+
+To encourage greater community participation, we propose to create the following roles while ensuring that ultimate decision-making authority remains with the community through on-chain voting:
+- MIP Author
+- MIP Facilitator
+- MIP Reviewer
+- Dev Team
+
+#### Clarifying Responsibilities of Existing Roles: 
+
+**MIP Author:** any community member can present a proposal and start the MIP process.
+
+Responsibilities: 
+- Articulate the proposed changes clearly and concisely.
+- Explain the expected effects of the proposed changes.
+- Encouraging community participation and engaging in discussions around the proposal.
+- Actively seek feedback from other community members.
+- Check alignment with the community.
+
+Functions:
+- Draft, edit and refine the MIP using the provided template.
+- Follow the instructions provided in the GitHub/MIPs documentation.
+- Collect and incorporate feedback from the community.
+- Document alternative opinions, the options considered and other discussed topics.
+- Champion the MIP through each stage of the process.
+- Designate, coordinate and oversee a Dev Team for the MIP implementation.
+
+**Dev Team:** core development team, contributors and any other active participants that are making updates, parameter changes, and upgrades to the protocol itself. 
+
+Responsibilities:
+- Review MIP’s implementation plan.
+- Execute the implementation plan of approved MIPs.
+
+Functions:
+- Assess the feasibility of the MIP’s implementation plan and provide feedback to the MIP Author.
+- Ensure correct execution of the implementation plan in a timely manner.
+- Communicate and explain any delays and other important decisions about the implementation.
+
+#### Introducing New Roles and Responsibilities:
+
+**MIP Facilitator:** Elected community member with a deep understanding of Mina Protocol’s governance to oversee that the MIP process is followed correctly.
+
+Responsibilities:
+- Ensure a smooth progression of MIPs through the process.
+- Facilitate cooperation and collaboration between all participants and ensure they follow the rules.
+- Assist and support MIP Authors as necessary.
+
+Functions:
+- Accept or reject a MIP submission based on the established approval criteria.
```

Comment:
It may not be a concern yet, but we may want to add an appeal process or a way to request a different facilitator. Being public/transparent with the process may be enough, but rejection is a powerful right and authors may need access to a second opinion in case of conflict.

Reply:
This is a valid point—thank you for raising it. While the MIP Upgrade already emphasizes transparency in the decision-making process, introducing an appeal mechanism could indeed enhance fairness and accountability.

That said, I personally believe this is a refinement best addressed in a future MIP once we have more practical experience with the current framework. This will allow us to design an appeal process based on real-world insights and the needs of the community. But I am happy to include this topic in future governance calls to hear more perspectives!